packages/@aws-cdk-testing/framework-integ/test/aws-elasticloadbalancingv2/test/integ.nlb.minimum-capacity-unit.ts

```diff
@@ -0,0 +1,32 @@
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import { Stack, aws_ec2 as ec2, aws_elasticloadbalancingv2 as elbv2, App } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+
+class TestStack extends Stack {
+  constructor(scope: Construct, id: string) {
+    super(scope, id);
+
+    const vpc = new ec2.Vpc(this, 'Stack', {
+      maxAzs: 1,
+    });
+    const lb = new elbv2.NetworkLoadBalancer(this, 'LB', {
+      vpc,
+      // minimumCapacityUnit is devided by number of availability zones
+      // In this case, 2750 / 1 = 2750
+      minimumCapacityUnit: 2750,
+    });
+    const listener = lb.addListener('Listener', { port: 80 });
+    const groupOne = new elbv2.NetworkTargetGroup(this, 'TargetGroup', {
+      vpc, port: 80, targetType: elbv2.TargetType.INSTANCE,
+    });
+
+    listener.addTargetGroups('Default', groupOne);
+  }
+}
+
+const app = new App();
+new IntegTest(app, 'MinimumCapacityUnitNlbInteg', {
+  testCases: [
+    new TestStack(app, 'MinimumCapacityUnitNlbStack'),
+  ],
```

Comment:
nit: any chance we can add assertions to this test to ensure the LCU is set properly

Reply:
That's weird, not sure when the command was added but I see the command existed even int AWS CLI V1. I think I'm good to approve without assertions since this is a fairly straight forward change. Thank you for checking though.